app/src/main/java/org/amfoss/templeapp/activities/ReadSingleData.java

```diff
@@ -115,7 +116,7 @@ protected Void doInBackground(Void... params) {
                 */
                 if (jsonObject != null) {
                     JSONObject user = jsonObject.getJSONObject(getString(R.string.user));
-                    name = user.getString(getString(R.string.name_));
+                    name = user.getString(getResources().getString(R.string.fetch_name));
```

Comment:
@gopikaharikumar is there any need to use `getResource()`. Can't you directly call `getString()` ?